src/models/__tests__/embeddingsList.test.ts

```diff
@@ -22,7 +22,7 @@ describe('EmbeddingsList', () => {
     expect(embeddingsList.model).toEqual(mockModel);
     expect(embeddingsList.data).toEqual(mockEmbeddings);
     expect(embeddingsList.usage).toEqual(mockUsage);
-    expect(embeddingsList.data.values).toEqual(mockEmbeddings.values);
+    expect(embeddingsList.data?.values).toEqual(mockEmbeddings.values);
```

Comment:
This soon won't be optional, though, right? A la this Slack convo? https://pinecone-io.slack.com/archives/C06HPKVKUFM/p1721324916903089

Might just want to make it with a `TODO` to come back to later, or at least make a high-priority Asana ticket

Reply:
(discussed offline): We're not changing the spec for now.